### PR TITLE
🏗 Make `gulp lint` output easier to read

### PR DESCRIPTION
**Pull request by rsimha**

### CODE REVIEW

File: build-system/eslint-rules/unused-private-field.js [hunk 1]
```diff
@@ -76,7 +76,7 @@ module.exports = function(context) {
         }
 
         context.report(node, [
-          `Unused private "${name}".`,
+          `Unused private "${name}".`.padEnd(80), // Padding for alignment
```
Comment (jridgewell) [inline build-system/eslint-rules/unused-private-field.js]: Hm?

Comment (rsimha) [reply build-system/eslint-rules/unused-private-field.js]: The `stylish` formatter of `eslint` uses the total length of the message.

https://github.com/eslint/eslint/blob/51132d6a0729b8e54326e9b8f5c8c1f487c8bd31/lib/formatters/stylish.js#L72-L77

**Comment by rsimha** [comment]
@alabiaga 

**Before:**
image


**After:**
image

**Pull request merged**